src/delagent/ui/admin-folder-delete.php

```diff
@@ -83,19 +83,24 @@ public function Output() {
     /* If this is a POST, then process the request. */
```

Comment:
I forgot much already, but I remember that you can port the plugins to the new rendering mechanism with a template. Depending on how much work this is, it's always nice to separate work/logic from the template